HelpMyStreet.Utils/HelpMyStreet.Utils/Models/UserPersonalDetails.cs

```diff
@@ -7,13 +7,23 @@ namespace HelpMyStreet.Utils.Models
     public class UserPersonalDetails
     {
         public User User { get; set; }
+
+        // Supplied in registration step 1
+        public string EmailAddress { get; set; }
+
+
+        // Supplied in registration step 2
         public string FirstName { get; set; }
         public string LastName { get; set; }
-        public string DisplayName { get; set; }
-        public DateTime? DateOfBirth { get; set; }
         public Address Address { get; set; }
-        public string EmailAddress { get; set; }
         public string MobilePhone { get; set; }
         public string OtherPhone { get; set; }
+
+        // Defaulted to FirstName following step 2
+        public string DisplayName { get; set; }
```

Comment:
This has been defined twice.

Reply:
What has?  We default DisplayName to the value of FirstName, but users can change it later.